rmgpy/cantherm/data/HSOO.yml

```diff
@@ -0,0 +1,43 @@
+InChI: InChI=1S/HO2S/c1-2-3/h3H
+InChI_Key: PWUAZNLOWMSNQX-UHFFFAOYSA-N
+SMILES: '[O]OS'
+adjacencyList: 'multiplicity 2
+
+  1 S u0 p2 c0 {2,S} {4,S}
+
+  2 O u0 p2 c0 {1,S} {3,S}
+
+  3 O u1 p2 c0 {2,S}
+
+  4 H u0 p0 c0 {1,S}
+
+  '
+atom_energies: ''
+author: alongd
+chemkin_thermo_string: "HSOO                    H   1S   1O   2     G    10.000  3000.000\
+  \  363.84      1\n 4.45295619E+00 8.05631845E-03-5.54113213E-06 1.76698187E-09-2.12266359E-13\
+  \    2\n 1.38487574E+04 8.17796073E+00 3.80904432E+00 2.01786111E-02-7.63088230E-05\
+  \    3\n 1.69530223E-07-1.41660098E-10 1.38622319E+04 1.01747051E+01           \
+  \        4\n"
+conformer: Conformer(E0=(115.143,'kJ/mol'), modes=[IdealGasTranslation(mass=(65.0667,'amu')),
+  NonlinearRotor(inertia=([91.5258,82.869,8.65696],'amu*angstrom^2'), symmetry=1),
+  HarmonicOscillator(frequencies=([266.165,512.727,890.356,1175.41,2529.35],'cm^-1')),
+  HinderedRotor(inertia=(4.80315,'amu*angstrom^2'), symmetry=1, fourier=([[0.305495,-0.899654,-0.432911,-0.0818798,-0.0102349],[0.000181646,0.000219011,0.000120536,1.95099e-05,-1.8673e-05]],'kJ/mol'))],
+  spinMultiplicity=2, opticalIsomers=1, mass=([31.9721,1.00783,15.9949,15.9949],'amu'),number=[16,1,8,8],
+  coordinates=([[-6.95052,3.36369,0.00013063],[-7.09087,5.91987,-0.000118519],[-3.50502,3.51303,0.000114712],[-2.45803,1.33214,0.000328714]],'angstroms'))
+datetime: 2018-06-15 11:32
+energy_transfer_model: SingleExponentialDown(alpha0=(0.5718,'kcal/mol'), T0=(300,'K'),
+  n=0.85)
+frequency_scale_factor: 0.975
+label: HSOO
+level_of_theory: CCSD(T)-F12a/cc-pVTZ-F12//B3LYP/6-311G(2d,d,p), rotors at B3LYP/6-311G(2d,pd)
+model_chemistry: CCSD(T)-F12/cc-pVTZ-F12
```

Comment:
I think the `level_of_theory` and `model_chemistry` may be a bit confusing/repetitive. This would require some refactoring of existing code, but you could consider renaming `model_chemistry` to `level_of_theory_energy`, then create two new attributes called `level_of_theory_geometry` and `level_of_theory_rotors`.

Reply:
I agree that it's a bit confusing this way, but I'm not sure that `model_chemistry` and `level_of_theory_energy` are interchangeable. Since `model_chemistry` is being used by Cantherm to apply the respective bond and atom corrections, users could specify a "close enough" `model_chemistry` to the actual `level_of_theory_energy` they used. I'm not saying it's a good thing to do, but might happen. I want to report here both, so first we know what real `level_of_theory_energy` was used, but also which type of corrections were implemented.

Perhaps a good approach to solve this confusion would be to only dump `model_chemistry` into the YAML file if it's different than the energy level of theory.